apps/settings/index.html

```diff
@@ -255,204 +255,204 @@
       -->
 
     <!-- Connectivity :: Wi-Fi :: Network Status -->
-    <section is="wifi-status" role="region" id="wifi-status" class="dialog"></section>
+    <section role="region" class="dialog" is="wifi-status" id="wifi-status"></section>
```

Comment:
make longer repeat section `<section role="region" ` to save some space after zip (its minor though)